config/nem.php

```diff
@@ -72,14 +72,16 @@
     */
     'nis' => [
         'primary' => [
+	        "protocol" => "http",
             "use_ssl" => false,
             "host" => "127.0.0.1",
             "port" => 7890,
             "endpoint" => "/",
         ],
         'testing' => [
+        	"protocol" => "http",
             "use_ssl" => false,
-            "host" => "go.nem.ninja",
+            "host" => "10.0.2.2",
```

Comment:
is it possible to revert this change? Not sure but I guess the 10.X IP is a local node of yours.